src/pages/UpdateRiddlePage.tsx

```diff
@@ -0,0 +1,19 @@
+import type { FC } from 'react';
+import { Navigate, useParams } from 'react-router-dom';
+
+import { UpsertRiddleForm } from '../components/forms/riddleUpsertForm/UpsertRiddleForm';
+import { useRiddleUpsert } from '../hooks/useRiddleUpsert';
+
+export const UpdateRiddlePage: FC = () => {
+	const { id } = useParams();
+
+	const riddle = useRiddleUpsert(id ?? '');
+	if (riddle === null) {
```

Comment:
Nestačilo by `!riddle`?

Reply:
Nestačilo, viz další podmínka: `riddle ?`.
Je potřeba odlišit čekání na to, než se data fetchnou, a situaci, kdy žádná data nejsou k dispozici.

Reply:
To by se ale asi mělo dít již v tom hooku (v ifu v něm, o kterém jsem psal v předchozím komentáři). Stejným způsobem je to uděláno i v `useRiddle<type>DetailFetch()`.

Reply:
Okay to asi dává smysl, podívám se, jak to je, a upravím podle toho.